
#include "util.h"
#include "data_struct.h"
#include "kandan_small.h"

#include "proto_kandan_small.pb.h"

void query_0_plan_4(oid_t param_id_0, kandan_small::PQuery0Result& qresult);


void query_1_plan_0(oid_t param_uid_0, kandan_small::PQuery1Result& qresult);


void query_2_plan_2(oid_t param_channel_id_0, kandan_small::PQuery2Result& qresult);


void query_3_plan_16(oid_t param_channel_id_0, kandan_small::PQuery3Result& qresult);


void query_4_plan_2(VarChar<64> param_name_0, kandan_small::PQuery4Result& qresult);


void query_5_plan_3(kandan_small::PQuery5Result& qresult);


void query_6_plan_0(LongString param_keyword_0, kandan_small::PQuery6Result& qresult);

